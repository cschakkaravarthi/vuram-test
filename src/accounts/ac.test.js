import React from 'react';
import { shallow } from 'enzyme';
import ac from '../accounts/ac';

test('should test Account component', () => {
 const wrapper = shallow(<ac />);
 expect(wrapper).toMatchSnapshot();
});