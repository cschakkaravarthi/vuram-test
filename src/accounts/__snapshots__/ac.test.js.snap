// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Account component 1`] = `<ac />`;
